File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/invitation_code_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class InvitationCodeForm:
    """Invitation code creation form"""
    code: str
    creator_user_id: int
    status: str = None
    expire_time: str
    usage_limit: int = None
    used_count: int = None

File: sdkwork-backend-sdk-go/types/invitation_code_form.go
package types

// Invitation code creation form
type InvitationCodeForm struct {
	Code string `json:"code"`
	CreatorUserId int `json:"creatorUserId"`
	Status string `json:"status"`
	ExpireTime string `json:"expireTime"`
	UsageLimit int `json:"usageLimit"`
	UsedCount int `json:"usedCount"`
}
